Assets/Scripts/Input/Player.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Script controls player character behaviour
+/// </summary>
+public class Player : MonoBehaviour
+{
+    #region Fields
+
+    public float runningVelocity;
+    public float fallVelocityLimit;
+    public float jumpVelocity;
+
+    public float yAccel;
+    private float yVelo = 0f;
+    private bool grounded = false;
+    private bool jumpingGrace = false;
+    public float jumpingGraceTime;
+    private bool isLanding = false;
+    private bool cielingCollision = false;
+    public float dropFromCielingVelocity;
+    
+    private float xVelo = 0f;
+
+    public float floatMargin;
+    public float wallMargin;
+
+    //ray tracing
+    public float horizRayMargin;
+    public float downRayMargin;
+    public float upRayMargin;
+    public float bottomRayMargin;
+    public float topRayMargin;
+    private float width;
+    private float height;
+    #endregion
+
+    #region Start, Update
+
+    private void Start()
+    {
+        height = transform.localScale.y;
+        width = transform.localScale.x;
+        Gamemaster gamemaster = FindObjectOfType<Gamemaster>();
```

Comment:
Use Gamemaster.Instance

Reply:
Done.
Also moved to Awake(), like in the Level script